src/main/resources/assets/computercraft/lua/rom/programs/http/webrun

```diff
@@ -0,0 +1,24 @@
+Arg = {...}
```

Comment:
All variable names should start with a lowercase (`Arg` -> `arg`). Also, there could be more than one argument, so it could be renamed `args`.